frontend/app/src/App.test.tsx

```diff
@@ -77,6 +77,8 @@ import {
 import { showDevelopmentOptions } from "./showDevelopmentOptions"
 import { App, log, Props } from "./App"
 
+type ViMock = ReturnType<typeof vi.fn>
```

Comment:
I think you can `import { MockInstance } from vitest` If you want the specific type. It might be Mock, but I think it's MockInstance.

Reply:
Resolved these using spy instead 👍🏼